pkgs/applications/misc/gcstar/default.nix

```diff
@@ -0,0 +1,76 @@
+{ stdenv
+, fetchFromGitLab
+, bash
+, perl
+, gtk2
+, perlPackages
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gcstar";
+  version = "1.7.2";
+
+  src = fetchFromGitLab {
+      owner = "Kerenoc";
+      repo = "GCstar";
+      rev = "v${version}";
+      sha256 = "1vqfff33sssvlvsva1dflggmwl00j5p64sn1669f9wrbvjkxgpv4";
+    };
+
+  perlDeps = [
+    perl
+    perlPackages.ArchiveZip
+    perlPackages.DateCalc
+    perlPackages.DateTimeFormatStrptime
+    perlPackages.Glib
+    perlPackages.Gtk2
+    perlPackages.GD
+    perlPackages.GDGraph
+    perlPackages.GDText
+    perlPackages.HTMLParser
+    perlPackages.JSON
+    perlPackages.ImageExifTool
+    perlPackages.librelative
+    perlPackages.LWPUserAgent
+    perlPackages.LWPProtocolHttps
+    perlPackages.MP3Info
+    perlPackages.MP3Tag
+    perlPackages.NetFreeDB
+    perlPackages.OggVorbisHeaderPurePerl
+    perlPackages.Pango
+    perlPackages.XMLSimple
+    perlPackages.XMLParser ];
+
+  nativeBuildInputs = [ wrapGAppsHook ];
+  propagatedBuildInputs = perlDeps;
+
+  installPhase = ''
+    runHook preInstall
+
+    cd gcstar
+
+    perl install --text --prefix=$out
+
+    runHook postInstall
+  '';
+
+  postFixup = ''
+
+    wrapProgram $out/bin/gcstar --prefix PERL5LIB : $PERL5LIB
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://gitlab.com/Kerenoc/GCstar";
+    description = "A general purpose collection manager";
```

Comment:
Maybe we should follow https://gitlab.com/Kerenoc/GCstar/-/blob/Test/gcstar/packages/GCstar_Packaging_Policy.txt

Reply:
I took the text from that file. How about displaying it under "Office" in the menus.... how can we control that?

Reply:
That should be the `Category` in the installed `desktop` file.

Reply:
The default desktop file has the "Category" office, so no patching is needed.